### ISSUE TITLE
Intermittent test failure - Importing records from a CSV file

### ISSUE THREAD

**Issue by maxkadel** [curationexperts-deprecated/tenejo-v1#322]
Example run - https://app.circleci.com/pipelines/github/curationexperts/tenejo/295/workflows/c330e9fd-e2e2-4cd3-a90d-92a3834e6585/jobs/435 

```
Importing records from a CSV file logged in as an admin user starts the import - spec.system.import_from_csv_spec
spec/system/import_from_csv_spec.rb
Failure/Error: click_on 'Start Import'

Net::ReadTimeout:
  Net::ReadTimeout with #<TCPSocket:(closed)>

[Screenshot]: tmp/screenshots/failures_r_spec_example_groups_importing_records_from_acsv_file_logged_in_as_an_admin_user_starts_the_import_378.png


./spec/system/import_from_csv_spec.rb:40:in `block (3 levels) in <top (required)>'
```

**Comment by mark-dce** [comment]
I think this is due to race/timing conditions in the Capybara matchers.

There's a good general overview of the issue here: https://medium.com/@ahmadhamza/i-learned-about-capybara-matchers-who-obey-wait-time-4a2d0b6df287 and here https://www.simplybusiness.co.uk/about-us/tech/2015/02/flaky-tests-and-capybara-best-practices/

If I scan the various gems the default wait times are all over the board - from 1 second to 20 (see grep output at bottom).  Rather than raising the overall timeout, it seems worth refactoring the test to: avoid navigating pages that aren't really about the thing under test; use capybara matchers that honor wait times; and change the wait time for individual tests where needed instead of upping the default for the whole test suite.

```
# grep -r "Capybara.default_max_wait_time" ./bundle/
./bundle/ruby/2.6.0/gems/browse-everything-0.15.1/spec/spec_helper.rb:Capybara.default_max_wait_time = ENV['TRAVIS'] ? 30 : 15
./bundle/ruby/2.6.0/gems/capybara-2.18.0/README.md:Capybara.default_max_wait_time = 5
./bundle/ruby/2.6.0/gems/capybara-2.18.0/README.md:  Capybara.default_max_wait_time = 2 # will not change the default_max_wait in my_session
./bundle/ruby/2.6.0/gems/capybara-2.18.0/lib/capybara/node/document_matchers.rb:      #   @option options [Numeric] :wait (Capybara.default_max_wait_time) Maximum time that Capybara will wait for title to eq/match given string/regexp argument
./bundle/ruby/2.6.0/gems/capybara-2.18.0/lib/capybara/spec/session/node_spec.rb:        Capybara.default_max_wait_time = 4
./bundle/ruby/2.6.0/gems/capybara-2.18.0/lib/capybara/spec/spec_helper.rb:        Capybara.default_max_wait_time = 1
./bundle/ruby/2.6.0/gems/capybara-2.18.0/spec/capybara_spec.rb:      Capybara.default_max_wait_time = 5
./bundle/ruby/2.6.0/gems/zizia-6.0.1/spec/dummy/spec/support/capybara.rb:Capybara.default_max_wait_time = 20

**Comment by mark-dce** [comment]
I came up with three alternatives to address the timeouts

1. remove the navigation part of the test
2. increase the wait time for the pages to render
3. increase the wait times globally

I've got branches with each option applied to the `mass_effect` branch and they all fix the problem for me locally and pass CI.  Once we choose one, I can cherry-pick it and make a PR against `main`

Here's diff's showing each option:

![image](https://user-images.githubusercontent.com/3064318/111821804-13572800-88b1-11eb-9c31-92a625c49bc1.png)


![image](https://user-images.githubusercontent.com/3064318/111821926-3550aa80-88b1-11eb-9bcb-73a6bbc9e9a5.png)


![image](https://user-images.githubusercontent.com/3064318/111822049-53b6a600-88b1-11eb-9695-607bd9427a08.png)
